app/src/main/java/org/asteroidos/sync/utils/AppInfoHelper.java

```diff
@@ -1,5 +1,24 @@
+/*
+ * AsteroidOSSync
+ * Copyright (c) 2023 AsteroidOS
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
 package org.asteroidos.sync.utils;
 
+// TODO Handle dubious copyright
```

Comment:
You can redistribute MIT code under GPL3. The sections are then also covered by the MIT licence, but do not require you to notify the user about their rights.  It also doesn't require you to redistribute your changes under the same licence. 
But I think you are right that either the licence should be more clearly noted or the code should change.

Can you please update comments regarding "dubious copyright" to clearly state what the issue is?

Reply:
Sure